src/sql/parts/connection/connectionDialog/connectionController.ts

```diff
@@ -54,6 +57,29 @@ export class ConnectionController implements IConnectionComponentController {
 		});
 	}
 
+	private updateDatabaseNames(serverName : string): Promise<string[]> {
+		let tempProfile = this._model;
+		tempProfile.authenticationType = Constants.integrated;
```

Comment:
>tempProfile.authenticationType = Constants.integrated; [](start = 2, length = 54)

why are we doing this?  Shouldn't we try to connect with what ever authentication type and credentials the user provided?

Reply:
SSDT only does it for Integrated Auth, because it does it automatically. If we're going the SSMS way, then it can support both auth types.